docs/it/authorization_endpoint.rst

```diff
@@ -132,10 +157,10 @@ L’oggetto request DEVE essere un token JWT firmato, secondo le modalità defin
  - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#rfc.section.2.4
  - https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests
 	 
-Claims
-++++++
+Claim
++++++
 
-Il parametro claims definisce gli attributi richiesti dal **RP**. Gli attributi SPID sono richiesti all’interno dell’elemento "userinfo", elencando gli attributi da richiedere come chiavi di oggetti JSON, i cui valori devono essere indicati come {"essential": true} o secondo le modalità definite dall’Agenzia per l’Italia Digitale. Non è possibile richiedere attributi SPID nell’id_token. Gli
+Il parametro claims definisce gli attributi richiesti dal **RP**. Gli attributi SPID sono richiesti all'interno dell'elemento "userinfo", elencando gli attributi da richiedere come chiavi di oggetti JSON, i cui valori devono essere indicati come {"essential": true} o secondo le modalità definite dall'Agenzia per l'Italia Digitale. Non è possibile richiedere attributi SPID nell'id_token. Gli
```

Comment:
rimuovere " o secondo le modalità definite dall'Agenzia per l'Italia Digitale" perché out of scope per CIE